cmake/Linters.cmake

```diff
@@ -71,7 +71,7 @@ if(NOT ${CLANG_FORMAT_CMD})
     set(CLANG_FORMAT_CMD clang-format)
     add_custom_target(
       clang-format
-      COMMAND xargs -a ${CLANG_FORMAT_TXT_FILE} -r -d '\;' ${CLANG_FORMAT_CMD} --verbose --Werror --dry-run -i --
+      COMMAND xargs -a ${CLANG_FORMAT_TXT_FILE} -r -d '\;' ${CLANG_FORMAT_CMD} --verbose --Werror --dry-run -i -- || echo ${RED}clang-format failed. Run \"make clang-format-fix-errors\" to fix the complaints.${COLOURRESET}
```

Comment:
Hmm, wouldn't this hide the return code of check? Therefore the test would succeed even if a problem was found?

Reply:
Thanks for catching this. Getting sloppy recently...